docs/podman-create.1.md

```diff
@@ -373,6 +373,10 @@ port to a random port on the host within an *ephemeral port range* defined by
 `/proc/sys/net/ipv4/ip_local_port_range`. To find the mapping between the host
 ports and the exposed ports, use `podman port`.
 
+**--quiet, -q**
+
+Suppress output information when pulling images
```

Comment:
nit, period at end here.  But that being said, there are a lot of lines not ending in a period.  FWIW, I think all lines should end in a period, comments or not, but especially man pages or other customer facing output.  I might take a hack through the pages next week and try to tighten that up.  Yeah my OCD is kicking in.

Reply:
Lets merge this and take care of man pages in different PR.